sycl/test/on-device/lit.cfg.py

```diff
@@ -191,8 +191,8 @@ def getDeviceCount(device_type):
 
 found_at_least_one_device = False
 
-cpu_run_substitute = "true"
-cpu_run_on_linux_substitute = "true "
+cpu_run_substitute = "echo "
+cpu_run_on_linux_substitute = "echo "
```

Comment:
Are arguments optional for `echo` on both windows and linux?

Reply:
They're not, but a) I didn't find the case when we have no args b) both can accept more than 1 argument